templates/base/storybook/.storybook/config.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { configure, addDecorator, addParameters } from '@storybook/react';
+import { DocsPage, DocsContainer } from '@storybook/addon-docs/blocks';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withDesign } from 'storybook-addon-designs';
+import { withA11y } from '@storybook/addon-a11y';
+import { ThemeProvider } from 'styled-components';
+import theme from '../../src/styles/themes/base/index';
+import '../../src/styles/cssIncludes';
+// import GlobalStyles from '../src/styles';
+
+addDecorator(withKnobs);
+addDecorator(withA11y);
+addDecorator(withDesign);
+
+addDecorator((storyFn, context) => (
+  <React.Fragment>
+    {/* <GlobalStyles /> */}
+    <ThemeProvider theme={theme}>{storyFn(context)}</ThemeProvider>
+  </React.Fragment>
+));
+
+// TODO: to be updated as per style guide later
+
+// const cssReq = require.context('!!raw-loader!../lib/styles/themes/base/tokens/', true, /.\.css$/);
+// const cssTokenFiles = cssReq
+//   .keys()
+//   .map((filename) => ({ filename, content: cssReq(filename).default }));
+
+// const svgIconsReq = require.context('!!raw-loader!../lib/styles/themes/base', true, /.\.svg$/);
+// const svgIconTokenFiles = svgIconsReq
+//   .keys()
+//   .map((filename) => ({ filename, content: svgIconsReq(filename).default }));
+
+addParameters({
+  options: {
+    name: `Universal React V2`,
```

Comment:
Universal React V2 --> This should be name of the project that gets created plus word Library

@hiteshparashar4 FYI

Reply:
@vinodloha - Currently, the name is` universal-react-v2`. Should we change it to `Universal React V2`?